src/mobile/time-picker/index.ts

```diff
@@ -53,12 +75,34 @@ export class MobileTimePicker extends KucBase {
     Object.assign(this, validProps);
   }
 
+  protected shouldUpdate(changedProperties: PropertyValues): boolean {
+    if (this._isFirstTimeInit) {
+      if (this.value === undefined) {
+        this.value = "";
+      }
+      this._isFirstTimeInit = false;
+    }
+    if (this.value === undefined || this.value === "") return true;
+    if (!validateTimeValue(this.value)) {
+      throwErrorAfterUpdateComplete(this, FORMAT_IS_NOT_VALID);
+      return false;
+    }
+    return true;
+  }
+
+  willUpdate(): void {
+    if (this.value === undefined || this.value === "") return;
+    this.value = timeValueConverter(this.value);
+  }
+
   update(changedProperties: PropertyValues) {
-    if (changedProperties.has("value") && this.value) {
-      if (!validateTimeValue(this.value)) {
-        throw new Error(FORMAT_IS_NOT_VALID);
+    if (changedProperties.has("value") && !this._isSelectError) {
+      if (this.value === undefined) {
+        this._inputValue = "";
```

Comment:
When INVALID_TIME_FORMAT error is displayed on UI, after set value undefined, the UI is empty, but error is still displayed.